services/user/interfaces/interfaces.go

```diff
@@ -98,3 +100,33 @@ func (handler *UserHttpHandler) AddVendorByAdminHandler(w http.ResponseWriter, r
 	}
 	pkg.EncodeResult(w, token, http.StatusOK)
 }
+
+// UpdateUserRecordHandler godoc
+// @Summary Update User Status
+// @Description The endpoint takes the request to update signed user records
+// @Tags User
+// @Accept json
+// @Produce json
+// @Param models.User body models.User true "update user record"
+// @Security BearerToken
+// @Success 200 {object} pkg.DefaultResponse
```

Comment:
Since there is not actual content from the response

```suggestion
// @Success 204 {object} pkg.DefaultResponse
```